go.work.sum

```diff
@@ -105,6 +105,7 @@ cloud.google.com/go/compute v1.23.0 h1:tP41Zoavr8ptEqaW6j+LQOnyBBhO7OkOMAGrgLopT
 cloud.google.com/go/compute v1.23.4/go.mod h1:/EJMj55asU6kAFnuZET8zqgwgJ9FvXWXOkkfQZa4ioI=
 cloud.google.com/go/compute v1.24.0/go.mod h1:kw1/T+h/+tK2LJK0wiPPx1intgdAM3j/g3hFDlscY40=
 cloud.google.com/go/compute v1.25.1 h1:ZRpHJedLtTpKgr3RV1Fx23NuaAEN1Zfx9hw1u4aJdjU=
+cloud.google.com/go/compute v1.25.1/go.mod h1:oopOIR53ly6viBYxaDhBfJwzUAxf1zE//uf3IB011ls=
```

Comment:
run `hack/update-go-workspace.sh`, I think these might be spurious and we might not be verifying this

Reply:
seems sticky even when i run `hack/update-go-workspace.sh` 

```
❯ go version
go: downloading go1.23.0 (darwin/arm64)
go version go1.23.0 darwin/arm64
```

Reply:
that's... so confusing, but... ok